doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2401,4 +2401,4 @@ Each set of labels is identified by a curationLabelSet name and a JSON Array of
 :AllowCustomTerms
 +++++++++++++++++
 
-By default, custom terms of data use and access can be specified after selecting "Custom Terms" from the License/DUA dropdown on the Terms tab. When ``:AllowCustomTerms`` is  set to ``false`` the "Custom Terms" item is not made available to the depositor.
+By default, custom terms of data use and access can be specified after selecting "Custom Terms" from the License/Data Use Agreement dropdown on the Terms tab. When ``:AllowCustomTerms`` is  set to ``false`` the "Custom Terms" item is not made available to the depositor.
```

Comment:
* The drop-down item must also be renamed to "Custom Terms of Use".
* I guess we'd better rename the setting `:AllowCustomTermsOfUse` as well.